templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,34 @@
 {
```

Comment:
<p>I think it's best to raise a <code>ValueError</code> here to make it consistent with similar validation checks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>